clustering/el/pom.xml

```diff
@@ -45,7 +45,8 @@
     </description>
 
     <modules>
-        <module>glassfish</module>
+		<!-- TODO uncomment when converted to jakarta.*
+        <module>glassfish</module>-->
```

Comment:
FYI, the glassfish module was only used for javax.el.* support.  jakarta.el.* support is exclusive to the expressly module.
I prepare a PR to this effect to today.

Reply:
Yeah; I just used the same code comment everywhere. Partly to make it easier to git grep for them later.

Reply:
Gotcha.